src/App.tsx

```diff
@@ -271,7 +272,7 @@ export function App() {
   });
 
   return (
-    <>
+    <DocuNinjaProvider>
```

Comment:
Unsure this is what we want. We're wrapping whole Invoice Ninja app (including self-hosted builds) with query to DocuNinja even without users opting into DocuNinja? Is this what you wanted to do?